_tutorials/new-component/create-a-new-component-part-2.md

```diff
@@ -150,27 +150,17 @@ First we will update our border colour using [o-colors](https://registry.origami
 
 ### Install Component Dependencies
 
-The first step is to install each component we want to use via [Bower](https://bower.io/) (a package manager like [NPM](https://www.npmjs.com/)). In order for Bower to find the components we will be installing, we need to tell it where to look. For that, we use a `.bowerrc` file. We recommend adding it to your home directory `~/.bowerrc`:
-<pre><code class="o-syntax-highlight--json">{
-	"registry": {
-		"search": [
-			"https://origami-bower-registry.ft.com",
-			"https://registry.bower.io"
-		]
-	}
-}</code></pre>
-
-With Bower configured, we can now run:
-<pre><code class="o-syntax-highlight--bash">bower install o-colors@^5.0.0 o-spacing@^2.0.0 o-typography@^6.0.0 o-buttons@^6.0.0 --save</code></pre>
+The first step is to install each component we want to use via [npm](https://npmjs.com/):
+<pre><code class="o-syntax-highlight--bash">npm install --save '@financial-times/o-colors@^5.0.0' '@financial-times/o-spacing@^2.0.0' '@financial-times/o-typography@^6.0.0' '@financial-times/o-buttons@^6.0.0'</code></pre>
 
-You should now have a `bower_components` directory with all the components we just installed. We can now make their Sass available for us to use with `@import` statements at the top of `main.scss`.
+You should now have a `node_modules` directory with all the components we just installed. We can now make their Sass available for us to use with `@import` statements at the top of `main.scss`.
 
 <pre><code class="o-syntax-highlight--diff">// main.scss
 
-+@import 'o-colors/main';
-+@import 'o-spacing/main';
-+@import 'o-typography/main';
-+@import 'o-buttons/main';
++@import '@financial-times/o-colors/main';
```

Comment:
If we require the index Sass file, we don't need `/main`?

Reply:
I was thinking we can keep both to make migration simpler as then current users would not need to rewrite all their `@import`s

Reply:
Shall we not use `main` in our docs?